src/snek5000/output/history_points.py

```diff
@@ -48,8 +48,9 @@ def __init__(self, output=None):
     def _complete_params_with_default(cls, params):
         params.output._set_child(
             "history_points",
-            attribs={"coords": None},
+            attribs={"coords": None, "write_interval": 100},
         )
+        params.output.history_points._record_nek_user_params({"write_interval": 1})
```

Comment:
Setting `params.output.write_interval` is fine, but setting `userParam01` to the same is case-specific and should not be here.

Reply:
I now see what you mean. For me, the principle was to put first the "base" userParams. Since history_points is built in Nek5000 and that our class is very general and could be used in all solvers using Snek, I though it would be better to use the first slot for it. But we can also use the last slot. Unfortunately, there is a bug for large value in Nek, so the last value that we can use seems to be 10 (and not 20).

Reply:
I see. Setting this as the `userParam10` is a good compromise. Thanks.